src/fides/api/schemas/connection_configuration/connection_secrets_google_cloud_mysql.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Optional
+
+from pydantic import Field
+
+from fides.api.schemas.base_class import NoValidationSchema
+from fides.api.schemas.connection_configuration.connection_secrets import (
+    ConnectionConfigSecretsSchema,
+)
+
+
+class GoogleCloudMySQLSchema(ConnectionConfigSecretsSchema):
```

Comment:
This should be using Google's `keyfile_creds` similar to how we do it in `connection_secrets_bigquery`. Take a look at option two in this doc for how to authenticate using a keyfile https://ethyca.atlassian.net/wiki/spaces/EN/pages/3029204995/2024-04-19+Google+Cloud+SQL+Integration#Option-2-(IAM-database-authentication)